docs/examples/models_orm_mode_kwargs.py

```diff
@@ -0,0 +1,31 @@
+import typing
+
+from pydantic import BaseModel
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+
+
+class MyModel(BaseModel):
```

Comment:
This is a pretty confusing example.

Better to do something like

```py
class MyModel(BaseModel):
    foo: str
    bar: int
    spam: bytes

    class Config:
        orm_mode = True


BaseModel = declarative_base()


class SQLModel(BaseModel):
    __tablename__ = 'my_table'
    id = sa.Column('id', sa.Integer, primary_key=True)
    foo = sa.Column('metadata', sa.String(32))
    bar = sa.Column('metadata', sa.Integer)

...

pydantic_model = MyModel.from_orm(sql_model, bar=123, spam=b'apple')
```
Or similar to show the behaviour.